js/main.js

```diff
@@ -82,19 +84,20 @@ function showNewNFT() {
         likes: null,
         dislikes: null
       };
-      if (data.ratingsInfo[collectionData.collectionName] === undefined) {
-        getCollectionPhotoURL(i);
+
+      if (!data.ratingsInfo[collectionData.collectionName]) {
+        throttle(getCollectionPhotoURL(randomInt), 3000);
```

Comment:
This `throttle()` isn't actually doing anything, because `getCollectionPhotoURL` is already being called. If your app is working as intended, then you can simply remove it.